WordPress/Classes/ViewRelated/Plans/PlanDetailViewController.swift

```diff
@@ -150,42 +148,47 @@ class PlanDetailViewController: UIViewController {
 
         return controller
     }
-    
+
     override func prefersStatusBarHidden() -> Bool {
         return true
     }
-    
+
     override func viewDidLoad() {
         super.viewDidLoad()
-        
+
         configureAppearance()
         configureTableView()
         populateHeader()
         updateNoResults()
     }
     
+    override func viewDidLayoutSubviews() {
+        super.viewDidLayoutSubviews()
+        
+        let size = headerView.systemLayoutSizeFittingSize(UILayoutFittingCompressedSize)
+        headerView.frame.size.height = size.height
+        tableView.tableHeaderView = headerView
+        view.layoutIfNeeded()
```

Comment:
Maybe add a comment explaining why this is needed. Also, and I really don't know, does this need to be recalculated for every layout, or could it be run once on `viewDidLoad`? Is there any chance this creates a loop since we're calling `layoutIfNeeded` from `viewDidLayoutSubviews`? Layout code isn't my strongest suit, but it _looks_ like something that could cause infinite recursion.